server/Dockerfile

```diff
@@ -1,26 +1,7 @@
-# 1: Build the exe
-FROM rust:1.42 as builder
-WORKDIR /usr/src
+FROM rust
```

Comment:
Building the container this way will most likely include the whole rust build system into the final image. Why delete this?